### PR TITLE
apps//userdashboard//projects: fix tiles clickable and shadow (multim…

### PR DESCRIPTION
**Pull request by khamui**
…odule projects and moderation dashboard).

Note: this is related to an issue fix, which was already closed.

### CODE REVIEW

File: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationProjects.jsx [hunk 1]
```diff
@@ -105,13 +111,7 @@ export default class ModerationProjects extends Component {
...
                       <span className="text-muted">{byText}{item.organisation}</span>
                       <h3 className="tile__title mb-2">
-                        <a
-                          href={item.moderation_detail_url}
-                          className="tile__link"
-                        >
-                          {item.num_unread_comments > 0 && <span className="text-info">• <span className="visually-hidden">{hasUnReadComments}</span></span>}
-                          {item.title}
-                        </a>
+                        {item.title}
```
Comment (goapunk) [inline apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationProjects.jsx:114]: one thing I noticed is that it is now read differently by the Screenreader (at least the chrome one). It doesn't read the title anymore when  tabbing through (so you need an extra step to read it). Not sure if that's an issue though. @philli-m any thoughts?

**Review by goapunk — APPROVED** [review]
apart from the comment about the SR looks good to me

File: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationProjects.jsx (hunk 1 above)
Comment (philli-m) [reply apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationProjects.jsx:114]: I can check on JAWS and NVDA as they are the most used

Comment (goapunk) [reply apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationProjects.jsx:114]: @khamui I used the one the chrome screen reader (which probably isn't very good). I can check with the other ones later.

Comment (goapunk) [reply apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationProjects.jsx:114]: NVDA only reads blank, I guess that's not ideal

Comment (khamui) [reply apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationProjects.jsx:122]: @goapunk @philli-m having the visually hidden h3 as a child worked finally..

File: apps/projects/templates/a4_candy_projects/includes/project_list_tile.html [hunk 2]
```diff
@@ -1,11 +1,14 @@
 {% load i18n project_tags a4_candy_project_tags contrib_tags thumbnail rules static %}
 
 <li class="tile tile--{{ orientation }}">
-
+    <a class="tile__link" aria-labelledby="tile-title-{{ object.pk }}" href="{{ url }}">
+        <h3 class="visually-hidden" id="tile-title-{{ object.pk }}">
+            {{ object.name }}
+        </h3>
+    </a>
```
Comment (philli-m) [inline apps/projects/templates/a4_candy_projects/includes/project_list_tile.html:8]: @khamui that took me too long to spot :p the link was just inside the if for tile image, so tiles with no image didn't have it

**Review by philli-m — APPROVED** [review]
nice!

**Pull request merged**